introduction.md

```diff
@@ -0,0 +1 @@
+Hello there, I'm Carlos from Argentina. Nice to meet you all :)
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.